packages/ui/src/styles/theme-chad.css

```diff
@@ -54,7 +54,6 @@ body.theme-chad {
   /* Buttons */
   --button--color: var(--text-600);
   --button--background-color: var(--primary-700);
-  --button--font-size: small;
```

Comment:
Won't this leave some undefined references?

Reply:
it will use the rule from the default theme